MatrixKit/Models/Account/MXKAccount.h

```diff
@@ -114,6 +125,14 @@ extern NSString *const kMXKAccountErrorDomain;
 @property (nonatomic,getter=isDisabled) BOOL disabled;
 
 /**
+ Register the MXKAccountOnCertificateChange block that will be used to handle certificate change during account use.
+ This block is nil by default, any new certificate is ignored.
```

Comment:
Okay - I'd probably comment it to say explicitly that it will abort the connection to the server.